<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>520</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons/icons.qrc">
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QVTKWidget" name="qvtkWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewCase"/>
    <addaction name="actionOpenBlockMeshDict"/>
    <addaction name="actionReOpenBlockMeshDict"/>
    <addaction name="actionOpenGeometry"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_hexBlocker"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_tool_bar"/>
    <addaction name="actionView_tool_box"/>
    <addaction name="actionViewVerticeEditor"/>
    <addaction name="separator"/>
    <addaction name="actionBlockVisibility"/>
    <addaction name="actionPatchVisibility"/>
    <addaction name="actionEdgeVisibility"/>
    <addaction name="actionVertIDVisibility"/>
    <addaction name="actionGeometryVisibility"/>
    <addaction name="separator"/>
    <addaction name="actionScaleFactor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenBlockMeshDict"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionCreateHexBlock"/>
   <addaction name="actionExtrudePatch"/>
   <addaction name="actionDeleteBlocks"/>
   <addaction name="separator"/>
   <addaction name="actionOpenMoveVerticesDialog"/>
   <addaction name="actionRotateVertices"/>
   <addaction name="separator"/>
   <addaction name="actionSetBCs"/>
   <addaction name="actionSetNumber"/>
   <addaction name="actionMergePatch"/>
   <addaction name="separator"/>
   <addaction name="actionDumpData"/>
   <addaction name="actionArbitraryTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/zoomOut.png</normaloff>:/Icons/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDumpData">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Dump data to std out</string>
   </property>
   <property name="toolTip">
    <string>Dump data to std out</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCreateHexBlock">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/createHexBlock.png</normaloff>:/Icons/createHexBlock.png</iconset>
   </property>
   <property name="text">
    <string>create HexBlock</string>
   </property>
   <property name="toolTip">
    <string>Create a new HexBlock</string>
   </property>
  </action>
  <action name="actionExtrudePatch">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/ExtrudeBlock.png</normaloff>:/Icons/ExtrudeBlock.png</iconset>
   </property>
   <property name="text">
    <string>extrude patch</string>
   </property>
   <property name="toolTip">
    <string>Extrude a patch</string>
   </property>
  </action>
  <action name="actionOpenMoveVerticesDialog">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/selectVertices.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>move vertices</string>
   </property>
   <property name="toolTip">
    <string>move vertices</string>
   </property>
  </action>
  <action name="actionSetBCs">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/SetBCs.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>setBCs</string>
   </property>
   <property name="toolTip">
    <string>Set BCs on patches</string>
   </property>
  </action>
  <action name="actionSetNumber">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/edgeProperties.png</normaloff>:/Icons/edgeProperties.png</iconset>
   </property>
   <property name="text">
    <string>setNumber</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;set number of cells on edges&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpenBlockMeshDict">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open BlockMeshDict</string>
   </property>
   <property name="toolTip">
    <string>Open a blockMeshDict</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenGeometry">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open Geometry</string>
   </property>
   <property name="toolTip">
    <string>Open a STL file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_hexBlocker">
   <property name="text">
    <string>About hexBlocker</string>
   </property>
  </action>
  <action name="actionView_tool_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View tool bar</string>
   </property>
  </action>
  <action name="actionView_tool_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View tool box</string>
   </property>
  </action>
  <action name="actionScaleFactor">
   <property name="text">
    <string>Set scale factor</string>
   </property>
   <property name="toolTip">
    <string>(convertToMeters)</string>
   </property>
  </action>
  <action name="actionArbitraryTest">
   <property name="text">
    <string>ArbitraryTest</string>
   </property>
   <property name="toolTip">
    <string>Test button for testing stuff</string>
   </property>
  </action>
  <action name="actionMergePatch">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/mergePatch.png</normaloff>:/Icons/mergePatch.png</iconset>
   </property>
   <property name="text">
    <string>mergePatch</string>
   </property>
   <property name="toolTip">
    <string>Merge two patches</string>
   </property>
  </action>

  <action name="actionRotateVertices">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/rotateVertices.png</normaloff>:/Icons/rotateVertices.png</iconset>
   </property>
   <property name="text">
    <string>rotateVertices</string>
   </property>
   <property name="toolTip">
    <string>Rotate a selection of vertices around a specified axis</string>
   </property>
  </action>

  <action name="actionSaveAs">
   <property name="text">
    <string>Save as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionReOpenBlockMeshDict">
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Reopen blockMeshDict</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNewCase">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDeleteBlocks">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/DeletHexBlock.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>delete blocks</string>
   </property>
   <property name="toolTip">
    <string>Delete one or several HexBlocks</string>
   </property>
  </action>
  <action name="actionBlockVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>block visibility</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionPatchVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>patch visibility</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEdgeVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>edge visibility</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionVertIDVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>vertice ID visibility</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionGeometryVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>geometry visibility</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionViewVerticeEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View VerticeEditor</string>
   </property>
   <property name="toolTip">
    <string>view table of vertices</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
